### ISSUE TITLE
User Modules permissions are not elevated even in unsafe mode.

### ISSUE THREAD

**Issue by LiEnby** [henkaku/henkaku#104]
Cant sceIoWrite() anywhere besides the apps own app0, savedata0, ux0:data/ etc just like a "safe" application would have those restrictions.

**Comment by DavisDev** [comment]
It is one of the reasons why plugins are used as intermediaries, which allow full access as:
- https://github.com/Rinnegatamante/kuio
- https://github.com/CelesteBlue-dev/PSVita-RE-tools#ioplus-01-02--by-dots-tb

**Comment by LiEnby** [comment]
yes, but it'd be nice if this was built into henkaku without needing other tools to make it work.
